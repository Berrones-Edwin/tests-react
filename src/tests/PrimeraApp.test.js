import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import React from 'react'
import PrimeraApp from '../PrimeraApp'

describe('Prueba in <PrimeraApp/>', () => {
  test('Debe de mostrar saludo', () => {
    const saludo = 'Hola, soy Goku'
    const wrapper = shallow(<PrimeraApp saludo={saludo} />)

    expect(wrapper).toMatchSnapshot()
  })
  test('Debe de mostrar subtitulo', () => {
    const saludo = 'Hola, soy Goku'
    const subtitulo = 'Soy un subtitulo'
    const wrapper = shallow(
      <PrimeraApp subtitulo={subtitulo} saludo={saludo} />
    )

    const textParagraph = wrapper.find('p').text()

    expect(textParagraph).toBe(subtitulo)



  })
})
