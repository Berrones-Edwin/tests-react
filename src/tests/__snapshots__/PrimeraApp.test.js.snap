// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba in <PrimeraApp/> Debe de mostrar saludo 1`] = `
<Fragment>
  <h1>
    Hola, soy Goku
    !!!
  </h1>
  <p>
    Soy un subtitulo
  </p>
</Fragment>
`;

